scripts/src/main/resources/scripts/command/ionic

```diff
@@ -14,6 +14,8 @@ if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
 TOOL_VERSION_COMMAND="ionic -v"
 # shellcheck source=scripts/functions
 source "$(dirname "${0}")"/../functions
+# shellcheck source=scripts/command/npm
+source "$(dirname "${0}")"/npm
```

Comment:
IMHO you can not source the script as this will execute its code as if it was pasted here what will cause undesired side-effects from the `npm` commandlet into the `ionic` commandlet.